firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/SessionReportingCoordinator.java

```diff
@@ -34,7 +34,7 @@
  * This class handles Crashlytics lifecycle events and manages capture, persistence, and sending of
  * reports to Firebase Crashlytics.
  */
-public class FirebaseCrashlyticsReportManager implements CrashlyticsLifecycleEvents {
+public class SessionReportingCoordinator implements CrashlyticsLifecycleEvents {
```

Comment:
Should we rename CrashlyticsLifecycleEvents to something less specific than SessionReportingCoordinator, e.g.,  SessionCoordinator, or maybe call the interface SessionReportingCoordinator and rename this class to something more specific about how it does it?

Reply:
In the long term, that's the plan, but this is one step in that direction. We'll come back to this.